ivy/functional/backends/numpy/experimental/activations.py

```diff
@@ -96,3 +96,17 @@ def elu(
 
 
 elu.support_native_out = True
+
+
+@_scalar_output_to_0d_array
+def celu(
+    x: np.ndarray,
+    /,
+    *,
+    alpha: float = 1.0,
+    complex_mode="jax",
+    out: Optional[np.ndarray] = None,
+) -> np.ndarray:
+    return (np.maximum(0, x) + alpha * np.expm1(np.minimum(0, x) / alpha)).astype(
```

Comment:
I think this should have been something like `(np.maximum(0, x) + np.minimum(0, alpha * np.expm1(x/alpha))).astype(x.dtype)` if you look at the docs for `celu` in [paddle](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/celu_en.html) or [torch](https://pytorch.org/docs/stable/generated/torch.nn.functional.celu.html) for example,

Reply:
Hi @hmahmood24 , thanks for the review. I also thought of going by the formula given in both torch and paddle docs, but checking jax's implementation suggests using the above approach as seen [here](https://jax.readthedocs.io/en/latest/_modules/jax/_src/nn/functions.html#celu).

Calling celu for paddle and torch backend will end up calling the provided backend.functional.nn.celu implementation, and for backend based on numpy (numpy, tensorflow, and jax), I decided to go with the implementation given by Jax in the attached link to have uniformity in their outputs.

Reply:
The implementation of torch.celu has seen [here, line 173](https://github.com/pytorch/pytorch/blob/a86727a06b53508df6cccf7c864596ed8e21fd79/torch/_refs/nn/functional/__init__.py#L157) is very similar to the implementation of Jax.